README.md

```diff
@@ -9,6 +9,8 @@ While it is possible to use the C# extension as a standalone extension, we highl
 1. Installing [C# Dev Kit][csdevkitextension] will automatically install this extension as a required dependency
 2. Open a folder/workspace that contains a C# project (.csproj) and a C# solution (.sln) and the extension will activate.
 
+Whether you install C# Dev Kit or just the C# extension, the [.NET Runtime Installer Tool extension](https://marketplace.visualstudio.com/items?itemName=ms-dotnettools.vscode-dotnet-runtime) will be installed as a dependency.
```

Comment:
nit: remove the numbering on the above lines or merge with numbering of this line.